index.js

```diff
@@ -235,12 +235,16 @@ async function fastifyStatic (fastify, opts) {
   }
 
   if (opts.decorateReply !== false) {
-    fastify.decorateReply('sendFile', function (filePath, rootPath) {
+    fastify.decorateReply('sendFile', function (filePath, rootPath, options = {}) {
```

Comment:
The default value is not necessary

```suggestion
    fastify.decorateReply('sendFile', function (filePath, rootPath, options) {
```